src/envo/envo-edit.owl

```diff
@@ -40289,6 +40295,92 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105116> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105116> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105116> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105116> "2021-05-07T18:06:28Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "entsorgte Angelausrüstung"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "equipo de pesca desechado"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "attrezzo di pesca scartato"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "weggeworfene Angelausrüstung"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "weggeworfene Fischerreiausrüstung"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105116> "weggeworfenes Angelzubehör"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105116> "envoPlastics")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105116> "In the literature, including GESAMP Report #99, this term is often referenced as part of abandoned, lost or otherwise discarded fishing gear (ALDFG)."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105116> "discarded fishing gear"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105116> <http://purl.obolibrary.org/obo/ENVO_06105115>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105117> (abandoned fishing gear)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.merriam-webster.com/dictionary/abandoned>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105117> "Fishing gear which has been purposefully left by the owner."@en)
```

Comment:
> Fishing gear which has been purposefully left by the owner.

The way it is phrased now would also include fishing gear that has e.g. been moved onto a ship to prepare for an expedition, left there in its designated space, and is ready to be used when needed. I'd argue that we wouldn't want that to fall under "abandoned fishing gear"

The previous definition was more clear:
> Fishing gear which has been purposefully left by the owner to fall into a state less "fuzzy".

@pbuttigieg @iwilkie How would you propose making the "state of disuse" more clear?

Reply:
Would this be clearer:

> Fishing gear which has been purposefully left by the owner and will not be returned to.